packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -55,23 +64,22 @@ import { REMOTE_TEAMS_APP_ID, REMOTE_MANIFEST } from "../../solution/fx-solution
 import AdmZip from "adm-zip";
 import * as fs from "fs-extra";
 
-export class AppStudioPluginImpl {
-  public async createApp(
-    appDefinition: IAppDefinition,
-    appStudioToken: string,
-    logProvider?: LogProvider,
-    colorIconContent?: string, // base64 encoded
-    outlineIconContent?: string // base64 encoded
-  ): Promise<IAppDefinition | undefined> {
-    return await AppStudioClient.createApp(
-      appDefinition,
-      appStudioToken,
-      logProvider,
-      colorIconContent,
-      outlineIconContent
-    );
-  }
+export enum PluginNames {
```

Comment:
import PluginNames, do not define it again.

Reply:
Thanks! I have moved PluginNames to /solution/constants.ts, and changed a few files that originally import PluginNames from solution.